config/samples/apps_v1alpha1_hstreamdb.yaml

```diff
@@ -11,10 +11,10 @@ metadata:
 spec:
   volumeClaimTemplate:
     spec:
-      storageClassName: 'hstore'
+      storageClassName: 'standard'
       resources:
         requests:
-          storage: 1G
+          storage: 0
```

Comment:
`0` is worked ? I'm not sure, I won't run it

Reply:
It meets no need PVC and set an `emptyDir` volume

Reply:
Cool, it will create emptyDir in StatefulSet ?